tools/bufr2zarr/bufr2zarr.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+# (C) Copyright 2024 NOAA/NWS/NCEP/EMC
+
+import sys
+import argparse
+
+import bufr
+from bufr.encoders import zarr
+
+def run(comm, data_path, mapping_path, output_path):
+    if comm.size() > 1:
+        container = bufr.Parser(data_path, mapping_path).parse(comm)
+        container.gather(comm)
+    else:
+        container = bufr.Parser(data_path, mapping_path).parse()
+
+    if comm.rank() == 0:
```

Comment:
@rmclaren No executable for bufr2zarr.  The data will be encoded through `zarr.Encoder` in Python, correct?  

```
dataset = next(iter(zarr.Encoder(YAML_PATH).encode(container, OUTPUT_PATH).values()))
```

Reply:
**bufr2zarr.py** is installed in the bin directory and can be treated just like **bufr2netcdf.x**. It even takes the same arguments. The zarr encoder is implemented in python as the zarr library has no c/c++ interface, that is true.